src/documents/tutorial-developerIntroduction/DeveloperIntroductionToInfusionFramework-ConcludingRemarks.md

```diff
@@ -0,0 +1,32 @@
+* All components support [events and inversion of control](DeveloperIntroductionToInfusionFramework-EventsAndInversionOfControl.md); these allow our programs to be built up of loosely coupled parts, and to manage sequencing through components observing and responding to their own events or the events of other components.
+* Components that need to track mutable data, state or content should be [model components](DeveloperIntroductionToInfusionFramework-ModelsAndModelComponents.md); model components can coordinate and synchronize their data with other model components, fire events when their models are changed, and take other actions to store and respond to state changes. Model components can use [model listeners](DeveloperIntroductionToInfusionFramework-ListeningToModelChanges.md) to respond to changes in model state.
+* Components that need to interact with the web page DOM to display content or interact with users are [view components](DeveloperIntroductionToInfusionFramework-ViewsAndViewComponents.md); view components are bound to specific DOM nodes when created and support the DOM Binder convention to avoid tying an implementation too tightly to specific markup.
+* Components can include [subcomponents, and use model relaying](DeveloperIntroductionToInfusionFramework-SubcomponentsAndModelRelaying.md) to keep state synchronized between different components in larger designs. Many kinds of model relays are possible aside from two-way synchronization.
+* As program designs evolve, Infusion's configuration-oriented components make it easier to [restructure a design](DeveloperIntroductionToInfusionFramework-RestructuringComponents.md) by splitting out functionality into multiple components and wiring them together through IoC references.
+* When it becomes clear two components have similar behaviour, Infusion's design helps in [refactoring to share functionality](DeveloperIntroductionToInfusionFramework-OverridingInvokersAndRefactoring.md) by overriding invokers and creating base grades.
+* Infusion has strong supports for [multimodal implementations](DeveloperIntroductionToInfusionFramework-ExtendingDesignsWithExistingComponents.md) that allow programs to adapt themselves to new contexts for input and presentation.
+* We can use [transformations as part of model relays](DeveloperIntroductionToInfusionFramework-TransformingModelRelays.md) to express more complex coordination of data between different parts of a program.
+
+## Where to Go Next?
+
+* Get [your own copy of Infusion from our GitHub repository](https://github.com/fluid-project/infusion).
+* Read about the [framework concepts](/FrameworkConcepts.md) to learn more about Infusion's design goals and overall philosophy.
+* Read [further documentation on Infusion](/), including many advanced topics not covered here such as [using Infusion in node.js](/NodeAPI.md), [advanced features of model transformation](/ModelTransformationAPI.md) and [unit testing](/jqUnit.md).
```

Comment:
All of the relative links in the "Where to Go Next?" section are broken. Probably because they are relative to the infusion docs and not the tutorials.